tests/unit/schemas/test_algorithm.py

```diff
@@ -71,27 +66,65 @@ def test_validate_rule(self):
                 blocking_keys=[],
                 evaluators={},
                 rule=rule,
-                cluster_ratio=0.5,
             )
         rule = "func:recordlinker.linking.matchers.feature_match_any"
         with pytest.raises(pydantic.ValidationError):
             AlgorithmPass(
                 blocking_keys=[],
                 evaluators={},
                 rule=rule,
-                cluster_ratio=0.5,
             )
         rule = "fn:recordlinker.linking.matchers.eval_perfect_match"
         AlgorithmPass(
             blocking_keys=[],
             evaluators={},
             rule=rule,
-            cluster_ratio=0.5,
         )
         rule = "recordlinker.linking.matchers.eval_perfect_match"
         AlgorithmPass(
             blocking_keys=[],
             evaluators={},
             rule=rule,
-            cluster_ratio=0.5,
         )
+
+
+class TestAlgorithm:
+    def test_validate_belongingness_ratio(self):
+        belongingness_ratio=(0.9, 0.75)
+        with pytest.raises(pydantic.ValidationError):
+            Algorithm(
+                label="label",
```

Comment:
@ericbuckley I initially did not pass a `label` arg because we set a [default](https://github.com/CDCgov/RecordLinker/blob/main/src/recordlinker/schemas/algorithm.py#L78), but then I got the below error. Passing a `label` resolved it. Any ideas why the default failed?
![Screenshot 2024-11-15 at 3 47 26 PM](https://github.com/user-attachments/assets/61409960-d3e9-4e6c-bdc0-cf0d68ca664c)

Reply:
@alhayward only `description` and `is_default` have default values (the empty string and False respectively).  The `label` attribute is assigned to be a [pydantic.Field](https://docs.pydantic.dev/latest/concepts/fields/) with a regex for validation.  It only has a default value if one is explicitly set when [initializing the field](https://docs.pydantic.dev/latest/concepts/fields/#default-values).